libraries/exoplayer/src/main/java/androidx/media3/exoplayer/drm/MediaDrmCallback.java

```diff
@@ -44,4 +46,15 @@ byte[] executeProvisionRequest(UUID uuid, ProvisionRequest request)
    * @throws MediaDrmCallbackException If an error occurred executing the request.
    */
   byte[] executeKeyRequest(UUID uuid, KeyRequest request) throws MediaDrmCallbackException;
+
+  /**
+   * Get the {@link LoadEventInfo} for the last executed request.
+   * <p></p>
+   * Valid after a call to {@link #executeKeyRequest(UUID, KeyRequest)} or
+   * {@link #executeProvisionRequest(UUID, ProvisionRequest)}, either contains
+   * the load event info for that request or null if no load is performed to execute the request
```

Comment:
nit - extend this wording to something like "or null if no load was performed or this implementation does not support reporting load info"